### PR TITLE
ADEN-4163 | call prebid on each page change on Mercury

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/run/mercury.run.js [hunk 1]
```diff
@@ -14,22 +14,20 @@ require([
...
-	customAdsLoader,
-	messageListener,
-	mercuryListener,
-	scrollHandler,
-	yavliTag,
-	geo,
-	instantGlobals,
-	win
-) {
+], function (adContext,
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/run/mercury.run.js:17]: MAJOR Function has 14 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [prebid.js#L12](https://github.com/Wikia/app/blob/91e5871711694e0d4f45dcd319f9f64f39908f27/extensions/wikia/AdEngine/js/lookup/prebid/prebid.js#L12): Function has 9 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [prebid.js#L12](https://github.com/Wikia/app/blob/66a1c169a3e57b64763fa29445d13b034e5a8609/extensions/wikia/AdEngine/js/lookup/prebid/prebid.js#L12): Function has 9 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [prebid.js#L12](https://github.com/Wikia/app/blob/9d649b0a99c072f6a397f9499e19d7a5a9b82f47/extensions/wikia/AdEngine/js/lookup/prebid/prebid.js#L12): Function has 9 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
- MAJOR 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [prebid.js#L12](https://github.com/Wikia/app/blob/febb8ea206fdda159bd40ee857dcbaec40a56736/extensions/wikia/AdEngine/js/lookup/prebid/prebid.js#L12): Function has 9 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Review by fraszczakszymon — APPROVED** [review]
LGTM

**Pull request merged**